docs/blog/_posts/2018-07-06-ninth-dotty-milestone-release.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.9.0-RC1
+author: Allan Renucci
+authorImg: /images/allan.jpg
+date: 2018-07-06
+---
+
+Today, we are excited to release Dotty version 0.9.0-RC1. This release serves as a technology
+preview that demonstrates new language features and the compiler supporting them.
+
+Dotty is the project name for technologies that are considered for inclusion in Scala 3. Scala has
```

Comment:
Dotty is the experimental compiler of Scala, considered to [stand](...) for the next Scala 3 compiler.
This release serves as a technology preview of new language features ready to experiment with by using Dotty.